.copier-answers.yml

```diff
@@ -0,0 +1,23 @@
+# Do NOT update manually; changes here will be overwritten by Copier
+_commit: v1.20
+_src_path: https://github.com/OCA/oca-addons-repo-template.git
+ci: GitHub
+convert_readme_fragments_to_markdown: false
+generate_requirements_txt: true
+github_check_license: true
+github_ci_extra_env: {}
+github_enable_codecov: false
+github_enable_makepot: true
+github_enable_stale_action: true
+github_enforce_dev_status_compatibility: false
+include_wkhtmltopdf: false
+odoo_test_flavor: OCB
```

Comment:
```suggestion
odoo_test_flavor: Both
```

Reply:
I tried to save the planet, but not today. It's fixed :-)

![](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExeGhhYWtlaHZ6OGp6eTltMmV6N203YXU1ODI1bW9ncGpiNWV6cjVwaCZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/1wPlBSbiRf6b6Dkk5C/giphy.gif)